pagopa/io-app
# fix(Firma con IO): [SFEQS-1258] Disable PDF annotation

## Short description
This PR disabled the PDF annotation feature in the FCI flow.

## List of changes proposed in this pull request
- Adds the `enableAnnotationRendering={false}` to the `Pdf` component.

## How to test
Replace a PDF in `assets/fci/pdf` with an editable PDF like [this one](https://biblus.acca.it/download/modello-pdf-editabile-spettanze-professionali/#:~:text=SCARICA%20MODELLO%20PDF%20EDITABILE%20SPETTANZE%20PROFESSIONALI) and start io-dev-api-server with the following configuration. Annotation should be disabled when rendering the PDF:
```
...
fci: {
      waitForSignatureCount: 1,
    },
...
```
